### PR TITLE
Cpuidle additions

### PR DESCRIPTION
**Pull request by bjackman**

### CODE REVIEW

File: devlib/module/cpuidle.py [hunk 1]
```diff
@@ -61,13 +61,25 @@ def name(self):
     @property
     @memoized
     def latency(self):
+        """State exit latency in uS"""
         return self.get('latency')
 
     @property
     @memoized
     def power(self):
+        """State power usage in mW"""
```
Comment (bjackman) [inline devlib/module/cpuidle.py]: AFAICT this value is actually garbage on ARM. Wasn't sure whether to mention that here...

Comment (setrofim) [inline devlib/module/cpuidle.py]: It's pobably worth adding as a ..note::

Comment (setrofim) [inline devlib/module/cpuidle.py]: Remove the verb "State" from the description.

File: devlib/module/cpuidle.py [hunk 2]
```diff
@@ -61,13 +61,25 @@ def name(self):
...
         return self.get('power')
 
+    @property
+    @memoized
+    def target_residency(self):
+        """State target residency in uS
```
Comment (setrofim) [inline devlib/module/cpuidle.py]: Remove the verb "State" from the description.

**Comment by bjackman** [comment]
Pushed a new version reworded without "state" and with a note about the power property probably being garbage.

**Pull request merged**